tensorflow/core/platform/profile_utils/cpu_utils.cc

```diff
@@ -33,6 +33,11 @@ static ICpuUtilsHelper* cpu_utils_helper_instance_ = nullptr;
      static const uint64 cpu_frequency = GetCycleCounterFrequencyImpl();
      return cpu_frequency;
 }
+#elif defined(__s390x__)
```

Comment:
The block here looks exactly same as the block above. Shouldn't we just add `|| defined(__s390x__)` to the end of the `if` statement above?